test/plugin/base_test.rb

```diff
@@ -1088,38 +1088,39 @@ def test_http_request_skip_setting_latency
     end
   end
 
-  def test_log_entry_trace_field
+  def verify_log_entry_field_key(log_entry_field, default_key, custom_key,
+                                 custom_key_config)
     setup_gce_metadata_stubs
     message = log_entry(0)
-    trace = 'projects/project-1/traces/1234567890abcdef1234567890abcdef'
+    sample_value = "sample #{log_entry_field} value"
```

Comment:
Seems like we should be doing this kind of test for all of the other fields (instead of `verify_subfields_from_record`, `verify_subfields_partial_from_record`, etc). Can we make the sample value a parameter as well, so we can eventualy apply this to `operation_key`, `source_location_key`, and `http_request_key`?

Reply:
Makes sense - I moved `sample_value` to be an argument to the method.